py/desispec/resolution.py

```diff
@@ -94,4 +105,32 @@ def to_fits_array(self):
         """
         return self.data
 
-#- (Unit tests moved to desispec.test.test_resolution)
\ No newline at end of file
+def _gauss_pix(x, mean=0.0, sigma=1.0):
+    """
+    Utility function to integrate Gaussian density within pixels
+
+    Args:
+        x (1D array): pixel centers
+        mean (float): mean of Gaussian
+        sigma (float): sigma of Gaussian
+
+    Returns:
+        array of integals of the Gaussian density in the pixels.
+
+    Note:
+        All pixels must be the same size
+    """
+    x = (np.asarray(x, dtype=float) - mean) / sigma
+    dx = x[1]-x[0]
+    if not np.allclose(np.diff(x), dx):
+        raise ValueError('all pixels must have the same size')
+
+    edges = np.concatenate([x-dx/2, x[-1:]+dx/2])
+    assert len(edges) == len(x)+1
+
+    y = erf(edges)
```

Comment:
I had to go hunting to find which erf you are using.  Since you are just using it once, a fully qualified name might be easier to read here.